index.html

```diff
@@ -114,4 +121,249 @@ <h3>2015</h3>
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <h2 class="connect-co">LET’S CONNECT</h2>
+
+        <ul class="social-media-icons">
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/github.png" alt="git" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/Linkedin.png" alt="linkedIn" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/angelite.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/twitter.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/medium.png" alt="image" />
+            </a>
+          </li>
+        </ul>
+        <div class="new-code">
+          <button type="submit" class="go-newmes">My resume</button>
+        </div>
+
+        <div class="skill">
+          <div class="event12">
+            <div class="event7">
+              <h3 class="languages">Languages</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+              </button>
+            </div>
+            <ul class="language-list">
+              <li class="event9">
+                <div class="lang">
+                  <img src="./images/Ellipse-1.png" alt="" />
+                  <p>JavaSript</p>
+                </div>
+              </li>
+              <li class="event10">
+                <div class="lang">
+                  <img src="./images/Ellipse-2.png" alt="" />
+                  <p>HTML</p>
+                </div>
+              </li>
+              <li class="event11">
+                <div class="lang">
+                  <img src="./images/Ellipse-3.png" alt="" />
+                  <p>CSS</p>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="event-skill">
+            <div class="event17">
+              <h3>Frameworks</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+              </button>
+            </div>
+            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            <div class="event18">
+              <h3>Skills</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+              </button>
+            </div>
+            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`, please kindly fix that 🙃 
- [ ] And then the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` 😁 😁
- [ ] You miss some images in your design and some lines, please kindly fix that 🙃 

Your design

<img width="282" alt="Screen Shot 2022-10-27 at 22 59 34" src="https://user-images.githubusercontent.com/74501523/198399009-6415d82b-b9f5-42d0-a449-ffe4b96f84bc.png">

Figma design

<img width="335" alt="Screen Shot 2022-10-27 at 23 00 05" src="https://user-images.githubusercontent.com/74501523/198398929-2cfddf59-db5a-4eae-9405-2c025c4e1962.png">